frontend/src/Page/Preparation.elm

```diff
@@ -599,6 +627,99 @@ uploadedIdentifierToLink identifier =
 
 
 
+-- ###################################################################
+-- CIP-100 Verification API
+-- ###################################################################
+
+
+{-| Convert IPFS URL to HTTPS gateway URL for the verification API.
+The CIP-100 verification API only accepts HTTPS URLs.
+-}
+ipfsToHttpsUrl : String -> String
+ipfsToHttpsUrl url =
```

Comment:
Is this manually done everywhere else? or is there already a function doing that at the other places where we do the conversion?